packages/dappmanager/src/daemons/telegramBot/telegramCommands.ts

```diff
@@ -0,0 +1,121 @@
+import TelegramBot from "node-telegram-bot-api";
+import * as db from "../../db";
+import { logs } from "../../logs";
+import { bold, buildTelegramMessage, url } from "./buildTelegramMessage";
+import { commandsList } from "./commandsList";
+
+/**
+ * Polls for commands and responds.
+ * Channel ID is not needed to response to messages
+ * @param telegramToken
+ */
+export async function telegramCommands(bot: TelegramBot): Promise<void> {
+  try {
+    bot.startPolling();
+    // POLLING ERRORS
+    // 1. EFATAL if error was fatal e.g. network error
+    // 2. EPARSE if response body could not be parsed
+    // 3. ETELEGRAM if error was returned from Telegram servers
+    // ETELEGRAM: 409 Conflict  =>  More than one bot instance polling
+    // ETELEGRAM: 404 Not Found => wrong token or not found
+    bot.on("polling_error", error => {
+      throw Error(`${error.name}: ${error.message}`);
```

Comment:
You can't throw an error in a callback like this. If you want the error to be visible you could log it